README.md

```diff
@@ -1,2 +1,36 @@
 # docker-base
-Bleeding edge tensorflow compiled to work with Python 3.7 and CUDA 9.2
+Bleeding edge Docker image with:
+
+- Ubuntu 18.04
+- Python 3.7b3
+- NVIDIA driver
+- CUDA 9.2.88
+- cuDNN 7.1.4.18
+- Tensorflow 1.9
+- Python packages (latest version to build date): numpy. scipy, keras, h5py, Pillow-simd, matplotlib, **jupyter**
+
+The default command launches Jupyter on port 8080.
```

Comment:
Maybe we should also say that the password for the  Jupyter notebook is "mallorca" (or remove it and tell people to put their own password on the line 163 of the Dockerfile.

Reply:
Fixed.